uber/common.py

```diff
@@ -32,7 +32,7 @@
 from collections import defaultdict, OrderedDict
 from os.path import abspath, dirname, exists, join
 from datetime import date, time, datetime, timedelta
-from threading import Thread, RLock, local, current_thread
+from threading import Thread, Timer, RLock, local, current_thread
```

Comment:
I'm finally starting to go through the dozens of PRs that have been opened in the past few weeks :)

Quick question here: are you actually using the ``Timer`` class anywhere?  I don't see any references to it in this PR and Sideboard has some more advanced facilities to accomplish the same thing if you need that sort of thing.